### PR TITLE
Fix tagging java-gradle-build-jib action

### PR DESCRIPTION
**Pull request by yordanovsstoyan**

### CODE REVIEW

File: actions/java-gradle-build-jib/action.yaml [hunk 1]
```diff
@@ -86,21 +82,34 @@ runs:
...
+            ${{ inputs.gradle-refresh-dependencies == 'true' && '--refresh-dependencies' || '' }} \
+            ${{ inputs.allow-insecure-registries == 'true' && '-Djib.allowInsecureRegistries=true' || '' }}
+        else
+          ./gradlew ${{ inputs.subproject && format('{0}:', inputs.subproject) || '' }}jib \
+            --info --stacktrace \
+            --image=${{ inputs.image-name }} \
+            -Djib.from.platforms=${{ inputs.platforms }} \
+            ${{ inputs.jib-from-image && format('-Djib.from.image={0}', inputs.jib-from-image) || '' }} \
+            ${{ inputs.class && format('-Djib.container.mainClass={0}', inputs.class) || '' }} \
+            -Djib.to.image=${{ inputs.full-image-name }} \
```
Comment (philipp94831) [inline actions/java-gradle-build-jib/action.yaml:109]: can you eliminate the if and rather do sth like `-[Djib.to](http://djib.to/).image=${{ image-name}}:{{ tag }} ${{ is-tag == 'true' && '-[Djib.to](http://djib.to/).tags=latest' || '' }}`? That way the other parameters stay the same

Comment (yordanovsstoyan) [reply actions/java-gradle-build-jib/action.yaml]: Is this ok for you 
```
-Djib.to.image=${{ inputs.full-image-name }}$(echo ${{ steps.meta.outputs.tags }} | sed 's/.*\(:.*\)/\1/') \
$([[ "${{ github.ref_type }}" == "tag" ]] && echo "-Djib.to.tags=latest") \
```

Comment (yordanovsstoyan) [reply actions/java-gradle-build-jib/action.yaml]: I just tested it and it works

File: actions/java-gradle-build-jib/action.yaml [hunk 2]
```diff
@@ -11,10 +11,6 @@ inputs:
     description: "Name of Docker image."
     required: false
     default: "${{ github.event.repository.name }}"
-  image-tags:
-    description: "Tag of Jib Image (comma separated list)"
-    required: false
-    default: "pipeline-${{ github.run_id }}-git-{{ sha }}"
   full-image-name:
     description: "Full name of image (registry/image:tag) If an image name is provided without a tag, 'latest' will be used. Be careful with registries that dont allow overwriting tags."
```
Comment (yannick-roeder) [inline actions/java-gradle-build-jib/action.yaml:15]:
```suggestion
    description: "Full name of image (registry/image). The image should be provided without a tag.
```

**Review by yannick-roeder — CHANGES_REQUESTED** [review]

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**